src/picongpu/include/simulation_defines/param/ionizerConfig.param

```diff
@@ -17,51 +17,158 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 
+/* @file ionizerConfig.param
+ *
+ * Ionization Model Configuration
+ *
+ * - None : no particle is ionized
+ * - BSIHydrogenLike : simple barrier suppression ionization
+ * - ADKLinPol : Ammosov-Delone-Krainov tunneling ionization (H-like)
+ *               -> linearly polarized lasers
+ * - ADKCircPol : Ammosov-Delone-Krainov tunneling ionization (H-like)
+ *                -> circularly polarized lasers
+ * - Keldysh : Keldysh ionization model
+ *
+ * Research and development:
+ * - BSIEffectiveZ : BSI taking electron shielding into account via an effective
+ *                   atomic number Z_eff
+ * - BSIStarkShifted : BSI for hydrogen-like atoms and ions considering the
+ *                     Stark upshift of ionization potentials
+ *
+ * Usage: Add flags to the list of particle flags that has the following structure
+ *
+ *        ionizer< particles::ionization::IonizationModel< Species2BCreated > >,
+ *        atomicNumbers< ionization::atomicNumbers::Element >,
+ *        effectiveNuclearCharge< ionization::effectiveNuclearCharge::ELEMENT_t >,
+ *        ionizationEnergies< ionization::energies::AU::ELEMENT_t >
+ */
```

Comment:
new syntax above for discussion, we might want to make all with the same `Element_t` spelling?